src/common/types/types.ts

```diff
@@ -7,3 +7,10 @@ export enum RelationSource {
   REQUEST = 'REQUEST',
   INVITATION = 'INVITATION',
 }
+
+export enum Status {
```

Comment:
```suggestion
export enum UserStatus {
```
What do you think of this? Because I think we might add some other status' in the future. Even if we don't it's probably best to be explicit.